.github/workflows/create-evaluation_reports.yml

```
@@ -0,0 +1,227 @@
+            */*.pksim5
+            */*_report.md
+        title: Update Evaluation Report for ${{ matrix['Snapshot name'] }}
+        body: |
+          ## Tools
+          ${{ needs.read_inputs.outputs.tools_table }}
+          ## All evaluated models
+          ${{ needs.read_inputs.outputs.models_table }}
+#        reviewers: ${{ github.actor }}
+
+
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (3 > 0)

(empty-lines)